internal/util/password/scramsha1_test.go

```diff
@@ -147,9 +147,11 @@ func TestSCRAMSHA1(t *testing.T) {
 
 		testutil.AssertNotEqual(t, doc1, doc2)
 
-		// salt is 18 bytes, but a value length increases ~33% when base64-encoded
```

Comment:
Both 16 and 18 could be encoded into 24 base64 symbols, so that test did not test much